Gemfile.lock

```diff
@@ -1,14 +1,14 @@
 GIT
-  remote: git://github.com/amatsuda/kaminari.git
-  revision: abbf93d557208ee1d0b612c612cd079f86ed54f4
+  remote: https://github.com/amatsuda/kaminari.git
+  revision: e83d6cebf69fd78e3c9830992209601fdc3e0f23
```

Comment:
Can you either manually change the revisions to the previous one, or lock it in the `Gemfile`?